### ISSUE TITLE
Create new layer for overlap areas with proper attributes

### ISSUE THREAD

**Issue by nilsnolde** [nilsnolde/ORStools#18]

**Comment by karussell** [comment]
Yes, especially the last layer should get transparent. Or a simpler trick first add the outer polygons so that the inner polygons are not hidden per default :)

**Comment by karussell** [comment]
Cool, thanks!

**Comment by lassegs** [comment]
@nilsnolde I'm curious as to how to produce overlaps now? Seperate polygons and then do clippings, or am I missing something.
